### ISSUE TITLE
the CI seems to be broken...

### ISSUE THREAD

**Issue by Rhialto** [PDP-10/klh10#56]
See https://github.com/PDP-10/klh10/runs/3000438023 :
```
Cirrus CI / main failed 2 hours ago in 3m 48s
Task Summary

Instruction main failed in 00:01
Details

✅ 00:01 clone
✅ 03:44 install
❌ 00:01 main

./autogen.sh
autom4te-2.69: need GNU m4 1.4 or later: /usr/local/bin/gm4
aclocal: error: autom4te failed with exit status: 1
autom4te-2.69: need GNU m4 1.4 or later: /usr/local/bin/gm4
autom4te-2.69: need GNU m4 1.4 or later: /usr/local/bin/gm4
autoheader-2.69: '/usr/local/bin/autom4te-2.69' failed with exit status: 1
Not running configure in the source directory.
mkdir tmp
cd tmp
../configure --enable-lights
/tmp/scripts9566c74d10037c4d7bbb0407d1e2c649.sh: ../configure: not found

Exit status: 127
```

**Comment by larsbrinkhoff** [comment]
I have seen this before, and I as far as I remember I haven't been able to fix it.

I think maybe a CI overhaul is needed.  GitHub Actions seems to work quite well.

**Comment by larsbrinkhoff** [comment]
Fixed by #63.

FreeBSD CI was dropped.
